compiler/rustc_passes/src/check_attr.rs

```diff
@@ -544,6 +544,37 @@ impl CheckAttrVisitor<'tcx> {
                         {
                             return false;
                         }
+                    } else if let Some(i_meta) = meta.meta_item() {
+                        if ![
+                            sym::cfg,
+                            sym::hidden,
+                            sym::html_favicon_url,
+                            sym::html_logo_url,
+                            sym::html_playground_url,
+                            sym::html_root_url,
```

Comment:
```suggestion
                            sym::html_no_source,
                            sym::html_root_url,
```

I guess we don't have tests for this attribute? Do you want to add one?

Reply:
Opening an issue.

Reply:
@GuillaumeGomez thanks - in the meantime, can you add it to the list of known attributes?